engines/wintermute/ad/ad_scene.cpp

```diff
@@ -2748,6 +2761,10 @@ bool AdScene::persist(BasePersistenceManager *persistMgr) {
 	persistMgr->transferPtr(TMEMBER_PTR(_viewport));
 	persistMgr->transferSint32(TMEMBER(_width));
 
+#ifdef ENABLE_WME3D
+	persistMgr->transferBool(TMEMBER(_2DPathfinding));
```

Comment:
indeed, we need to keep save games compatible with 2d

Reply:
That would mean, that this should be put into, #ifdef ENABLE_WINTERMUTE, right? Although this then would break compability with ScummVM savegames, which might still happen anyways because of the new 3d classes, as somaen pointed out somewhere else already.